jsonlines/jsonlines.py

```diff
@@ -275,15 +273,15 @@ def write(self, obj):
         # the serialised structure can be encoded using ASCII only, so
```

Comment:
this comment is no longer relevant since this change is about dropping python 2 altogether